py/images/builders/base.py

```diff
@@ -77,6 +77,11 @@ def open_target(self, image):
                     yield target
 
     def prepare_chroot(self, target):
+        apt_conf_liquid = target.mount_point / 'etc/apt/apt.conf.d/liquid'
```

Comment:
`liquid@liquid:~$ ls /etc/apt/apt.conf.d/` says: 
```01-vendor-ubuntu  01autoremove  01autoremove-kernels  01autoremove-postgresql  10periodic  15update-stamp  20archive  70debconf  99update-notifier```

So I'd go with naming our file `999-liquid-investigations`, even if `'l' > '9'` in ASCII.

Reply:
I don't think we need to worry about the order, we're the only ones to set those APT options.